### PR TITLE
Add Jswap Adapter

### PR DESCRIPTION
**Pull request by Lam-Jswap**
##### Twitter Link:
https://twitter.com/Jswap_Finance

##### List of audit links if any:
- https://www.slowmist.com/en/security-audit-certificate.html?id=928799684ad96ef4ed4b0c0fb12a5fae085456f874b19dc4300195b32a5a1431
- https://www.slowmist.com/en/security-audit-certificate.html?id=e18f32468d1c52dc0682fbf1c1137df933fabb09526b124c84cd391fce7149bb

##### Website Link:
https://app.jswap.finance

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
[logo](https://app.jswap.finance/logo_200x200.png)

##### Current TVL:
$50,070,435.18

##### Chain:
OKExChain

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
TODO

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
TODO

##### Short Description (to be shown on DefiLlama):
Jswap.Finance is the community-driven multi-chain compatible Dex and assets management protocol. 

##### Token address and ticker if any:
JF: `0x5fAc926Bf1e638944BB16fb5B787B5bA4BC85b0A`

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
None

##### forkedFrom (Does your project originate from another project):
None

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL counts the liquidity of Jswap DEX and staking counts the tokens and the JF in dividend pool that has been staked. Data is pulled from:"https://api.jswap.finance/stats/oec/tvl"

### CODE REVIEW

File: projects/jswap-finance/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const { fetchURL } = require('../helper/utils')
+const { getChainTvl } = require('../helper/getUniSubgraphTvl')
+const { staking } = require('../helper/staking')
+const sdk = require('@defillama/sdk')
+const BigNumber = require("bignumber.js")
+
+async function oecTvl(timestamp, ethBlock, chainBlocks) {
+    const response = await fetchURL('https://api.jswap.finance/stats/oec/tvl')
+
+    const liquidityTvl = new BigNumber(response.data.totalLiquidityUSD)
```
Comment (0xngmi) [inline projects/jswap-finance/index.js:10]: What does each of these numbers represent?

Comment (Lam-Jswap) [reply projects/jswap-finance/index.js:10]: - totalLiquidityUSD, total value of liquidity
- totalTokenStakedUSD, total value of single token staked
- totalDividendStakedUSD, total value of dividend staked

Comment (0xngmi) [reply projects/jswap-finance/index.js:10]: > total value of dividend staked

could you elaborate on this? what are staked dividends?

Comment (Lam-Jswap) [reply projects/jswap-finance/index.js:10]: Sure.
The platform token - `JF` will be staked into the dividends pool.
And users will get the dividend income (that from the DEX fee of platform) by proportion every day.

**Pull request merged**